auto_nag/scripts/topcrash_bad_severity.py

```diff
@@ -2,18 +2,48 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from auto_nag import utils
 from auto_nag.bzcleaner import BzCleaner
 
 
 class TopcrashBadSeverity(BzCleaner):
+    def __init__(self):
+        super(TopcrashBadSeverity, self).__init__()
+        self.extra_ni = {}
+
     def description(self):
         return "Bugs with topcrash keyword but incorrect severity"
 
     def ignore_date(self):
         return True
 
+    def get_mail_to_auto_ni(self, bug):
+        for field in ["assigned_to", "triage_owner"]:
+            person = bug.get(field, "")
+            if not utils.is_no_assignee(person):
+                return {"mail": person, "nickname": bug[f"{field}_detail"]["nick"]}
+
+        return None
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
```

Comment:
Nit: no need to store it in a varabile since you're just using it once in the line right after this

Reply:
This is just for consistency with the rest of the tools; almost all tools cast the bug ID in this way (35/36 cases). In most cases, the variable is used only once.

I filed issue #1432 to drop the need to do this typecasting. We should then drop the variable and the typecasting from most tools.